### PR TITLE
fixed change password param name

### PR DESCRIPTION
**Pull request by yneves**

### CODE REVIEW

File: src/redux/actions/auth/changePassword.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ const authPasswordProgress = createAction(AUTH_PASSWORD_PROGRESS);
 export default (currentPassword, newPassword, confirmPassword) => (dispatch, getState) => {
   if (newPassword === confirmPassword) {
     const { auth: { email } } = getState();
-    const params = { loginEmail: email, oldPassword: currentPassword, newPassword };
+    const params = { loginEmail: email, currentPassword, newPassword };
```
Comment (ntgn81) [inline src/redux/actions/auth/changePassword.js]: Can just change param name to oldPassword and use object shorthand here. Save some bytes, man! :space_invader:

Comment (yneves) [inline src/redux/actions/auth/changePassword.js]: lol i think you didn't look carefully, it's kind of what's done there

Comment (yneves) [inline src/redux/actions/auth/changePassword.js]: I checked the name in API is actually currentPassword, not oldPassword.

Comment (ntgn81) [inline src/redux/actions/auth/changePassword.js]: lol :+1:

**Pull request merged**